docs/csharp/tutorials/working-with-linq.md

```diff
@@ -257,8 +257,9 @@ Briefly, lazy evaluation states that the evaluation of a statement is not perfor
 
 Remember that we generated the original deck using a LINQ query. Each shuffle is generated by performing three LINQ queries on the previous deck. All these are performed lazily. That also means they are performed again each time the sequence is requested. By the time you get to the 52nd iteration, you're regenerating the original deck many, many times. Let's write a log to demonstrate this behavior. Then, you'll fix it.
 
-Here's a log method that can be appended to any query to mark that the query executed.
+In your `Extensions.cs` file, type in or copy the method below. This new extension method will create a new file called `debug.log` within your project's directory and record what query is currently being executed to the log file. This extension method can be appended to any query to mark that the query executed.
```

Comment:
- "new extension method" --> "extension method" (to eliminate the duplication of "new")
- nit: will create --> creates
- nit: project's directory --> project directory
- record --> records

Reply:
All of this sounds good to me, I'll make these changes as well.